packages/react-vapor/src/components/form/Form.tsx

```diff
@@ -7,11 +7,12 @@ export interface IFormProps {
     title?: string;
     className?: string;
     mods?: FormMods | FormMods[];
+    noMargin?: boolean;

```

Comment:
At first I thought it would be better to add this in the `mods`, but you'd need a bit of refactor

Reply:
I thought about it too. I don't have any preference so I went with the option that keeps it simple.